src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -436,6 +438,24 @@ unsigned uct_rc_verbs_iface_post_recv_always(uct_rc_verbs_iface_t *iface, unsign
     return count;
 }
 
+static void uct_rc_verbs_iface_vfs_refresh(uct_iface_h tl_iface)
+{
+    uct_rc_verbs_iface_t *iface = ucs_derived_of(tl_iface, uct_rc_verbs_iface_t);
+    uct_rc_verbs_ep_t *ep;
+
+    /* Add iface resources */
+    uct_rc_iface_vfs_populate(&iface->super);
+
+    /* Add objects for EPs */
+    ucs_list_for_each(ep, &iface->super.ep_list, super.list) {
+        ucs_vfs_obj_add_dir(iface, ep, "ep/%p", ep);
```

Comment:
may be add `eps` dir, so it is possible to check iface files without adding eps?

Reply:
i follow the same path names as for worker/iface, i.e. it is `uct/worker/.../iface/.../ep/.../`